app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -103,6 +102,13 @@ class WireActivityViewModel @Inject constructor(
                         appLogger.i("IncomingCall deepLink launched from the history")
                     }
                 }
+                is DeepLinkResult.OpenConversation -> {
+                    if (!isLaunchedFromHistory(intent)) {
+                        navigationArguments.put(OPEN_CONVERSATION_ID_ARG, result.conversationsId)
+                    } else {
+                        appLogger.i("OpenConversation deepLink launched from the history")
+                    }
+                }
```

Comment:
to make it more readable I suggest to flip the if like:
```suggestion
                is DeepLinkResult.OpenConversation -> {
                    if (isLaunchedFromHistory(intent)) {
                    appLogger.i("OpenConversation deepLink launched from the history")
                    } else {
                        navigationArguments.put(OPEN_CONVERSATION_ID_ARG, result.conversationsId)
                    }
                }
```